sirenity/code/handler.py

```diff
@@ -0,0 +1,43 @@
+import ast
+import os
+
+allowed_modules = ["math"]
```

Comment:
Capital letters for constants
```suggestion
ALLOWED_MODULES  = ["math"]
```